Adds example app list to readme

AdamBrodzinski opened the pull request:
I wasn't sure if this was even wanted but thought it was easier to PR without opening an issue. Would you like to add an examples section to the readme since everything is so new/experimental? 

Feel free to close this if that's bloating the readme.

Thanks!

hamiltop commented:
Maybe a wiki page for examples and link it from the readme?

On Mon, Oct 19, 2015, 4:11 PM Adam Brodzinski <notifications@github.com>
wrote:

> I wasn't sure if this was even wanted but thought it was easier to PR
> without opening an issue. Would you like to add an examples section to the
> readme since everything is so new/experimental?
>
> Feel free to close this if that's bloating the readme.
>
> Thanks!
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/hamiltop/rethinkdb-elixir/pull/71
> Commit Summary
>
>    - adds example app list to readme
>
> File Changes
>
>    - *M* README.md
>    <https://github.com/hamiltop/rethinkdb-elixir/pull/71/files#diff-0>
>    (5)
>
> Patch Links:
>
>    - https://github.com/hamiltop/rethinkdb-elixir/pull/71.patch
>    - https://github.com/hamiltop/rethinkdb-elixir/pull/71.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hamiltop/rethinkdb-elixir/pull/71>.
>

AdamBrodzinski commented:
Ah yea that's a much better idea! I was thinking that listing them directly in the readme felt cluttered. Just added a new wiki page for examples and added a new commit linking to that page.
